erpnext/setup/doctype/transaction_deletion_record/transaction_deletion_record.py

```
@@ -133,10 +254,258 @@ def before_save(self):
+			return {
+				"child_doctypes": "",
+				"document_count": 0,
+				"error": _("{0} is a child table and will be deleted automatically with its parent").format(
+					doctype_name
+				),
+			}
+
+		try:
+			# Use helper method to get deletion details
+			return self._get_to_delete_row_infos(doctype_name, company_field=None, company=company)
+		except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)